src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -109,6 +117,16 @@ public function queryAction(Request $request)
         );
     }
 
+    private function resolveIndexPlaceholders($value)
+    {
+        $resolver = $this->container->get('twig');
```

Comment:
Thought that'd be necessary to use multiple value. But thinking now again about it possibly not ;)
I'll change it